stacks/openshift/application/oidc-authservice-for-appid/README.md

```diff
@@ -5,6 +5,7 @@
 * Provisioning an [AppID](https://cloud.ibm.com/catalog/services/app-id) 
 instance from IBM Cloud. It can start with the _Lite_ plan, but will need the
 _Graduated tier_ once you need more than 1000 authentication events per month.
+* FQDN of OpenShift Route of istio ingress gateway.
```

Comment:
would it help to have a command to retrieve the route fqdn instead?

Reply:
is that ok I add something like `kubectl get route ...` later in the actual doc in another PR? don't have any OpenShift in hand now.

Reply:
k